examples/ropsten/contracts/RopstenConsumerBase.sol

```diff
@@ -31,15 +27,14 @@ contract RopstenConsumer is Chainlinked, Ownable {
   );
 
   constructor() Ownable() public {
-    setLinkToken(ROPSTEN_LINK_ADDRESS);
-    setOracle(ROPSTEN_ORACLE_ADDRESS);
+    newChainlinkWithENS(ROPSTEN_ENS, ROPSTEN_CHAINLINK_ENS);
   }
 
-  function requestEthereumPrice(string _currency) 
+  function requestEthereumPrice(string _jobId, string _currency) 
```

Comment:
Why not have these functions accept _jobId as bytes32 instead of string? Seems simpler than converting.

Reply:
Because when using a tool like Remix, you would have to [convert](https://www.rapidtables.com/convert/number/ascii-hex-bin-dec-converter.html) your input to a `bytes32` value first, and that's not very user-friendly for what should be a simple example.